packages/appstore/src/components/add-more-wallets/__test__/add-more-wallets.spec.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import AddMoreWallets from '../add-more-wallets';
+import { APIProvider } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+jest.mock('../wallet-add-card', () => {
+    const AddWalletCard = () => <div data-testid='dt-add-wallet-card'>AddWalletCard</div>;
+    return AddWalletCard;
+});
+
+jest.mock('../carousel-container', () => {
+    const CarouselContainer = ({ children }: { children: React.ReactNode }) => (
+        <div data-testid='dt-carousel-container'>{children}</div>
+    );
+    return CarouselContainer;
+});
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn((name: string) => {
+        if (name === 'authorize') {
+            return {
+                data: {
+                    authorize: {
+                        account_list: [
+                            { account_category: 'wallet', landing_company_name: 'svg', is_virtual: 0, currency: 'USD' },
+                        ],
+                        landing_company_name: 'svg',
+                    },
+                },
+            };
+        }
+
+        if (name === 'get_account_types') {
+            return {
+                data: {
+                    get_account_types: {
+                        wallet: {
+                            crypto: {
+                                currencies: ['BTC', 'ETH'],
+                            },
+                            doughflow: {
+                                currencies: ['USD', 'EUR'],
+                            },
+                        },
+                    },
+                },
+            };
+        }
+
+        return { data: undefined };
+    }),
+}));
+
+describe('AddMoreWallets', () => {
+    it('should render the component without errors', () => {
+        const mock = mockStore({
+            client: {
+                loginid: 'CRW909900',
+                accounts: {
+                    CRW909900: {
+                        token: '12345',
+                    },
+                },
+                is_crypto: (currency: string) => currency === 'BTC',
+            },
+        });
```

Comment:
I see this is reuse in multiple test case can we move this to 1 const mock_props?

Reply:
It's better to isolate the unit test files and don't rely on out sourced mocked props. This way a small change in one test will not affect the others.